content/tutorials/manage-account-settings/assign-a-regular-key-pair.md

```diff
@@ -394,6 +394,8 @@ Note that the response contains a `hash` of the transaction, which you can use t
 
 ## 3. Verify the Regular Key Pair
 
+At this point, the regular key pair is assigned to your account and you should be able to send transactions using the regular key pair. However, it is important to verify that you did not make a mistake before you take any additional steps such as [disabling the master key pair](disable-master-key-pair.html).
+
 To verify that your account has the regular key pair set correctly, submit an [AccountSet transaction][] from your account, signing it with the regular private key you assigned to your account in step 2.
 
 As discussed in step 2, transmitting your master private key is dangerous. It is equally risky to transmit your regular private key. Therefore, we'll complete this transaction in two steps to keep transaction signing separate from transaction submission to the network.
```

Comment:
"omplete this transaction in two steps " - I think the two separate steps are Sign your transaction and Submit your transaction, but it took a few extra hops to figure that out. So I might suggest:
Therefore, we'll complete this transaction with two separate steps--signing your transaction and submitting your transaction--to keep transaction signing separate from transaction submission to the network.

Reply:
I've rephrased both instances of this to link to the tutorial for setting up secure signing instead. (That tutorial didn't exist when the "Assign a Regular Key Pair" tutorial was originally written.)